draft-ietf-moq-requirements.md

```diff
@@ -675,6 +680,11 @@ groups and just wait for the next group when needed.
 
 Of course, having longer blocks create other issues. Realtime conferences also need to accomodate the occasional occasional late comer, or the disconnected user who want to resynchronize after a network event. This drives a need for synchronization "between access points". For example, rather than waiting for 30 seconds before connecting, the user might quickly download the "key" frames of the past 30 seconds and replay them in order to "synchronize" the video decoder.
 
+### Media Insertion and Redirection
+
+In all use cases it may be necessary for consumers to switch sources based on either triggering provided within a stream of media, or externally in particular to support the insertion of advertising or promotions, but may be useful for other requirements where an upstream encoder or producer is unable to do. Relays may also require consumers to relocate their subscriptions of media in order to facilitate this.
+
+In order to make seemless switching from one source to another and avoid stalling of playback, consumers must be able to receive media from sources concurrently. Support for common in-band signalling formats such as {{SCTE-35}} should also be supported.
```

Comment:
> In order to make seemless switching from one source to another and avoid stalling of playback, consumers must be able to receive media from sources concurrently.

Concurrent reception is not required to enable seamless switching. Since players have a forward buffer, the downloader can retrieve content from the new source and append it to the tail of the source buffer while the decoder is still processing the head.

Reply:
In general I find this section a bit unclear. Here's how I would phrase it

### Media Splicing
Some use-cases, such as advertising insertion, require the splicing together of different content sources while maintaining the perception of seamless playback by the end-user. These splice points may be described and triggered by in-band markers, such as {SCTE-35}, or may be the consequence of application-provided data. The splicing of the content may occur on the server-side or  the client-side and both locations should by supported by moq-transport.

Reply:
Thanks for the feedback. When I talk abut "use cases" it specifically refers the ones we define earlier in the document. Also I chose very deliberately the words "insertion and redirection" because of their functional differences; insertion being akin to splicing where media at the same source is changed and decoder needs to be aware of changes in parameters/timecode etc, and redirection is telling the consumer to go play media from a different source. I think that distinction is kind of important to have, however I'll try to make some test that is clearer.